Fix:
tclbasic.c:711:18: error: implicit declaration of function 'Tcl_CheckForSignal' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        result = Tcl_CheckForSignal (interp, result);
                 ^
--- src/tclx/ucbsrc/tclbasic.sed.orig	2018-06-29 04:19:31.000000000 -0500
+++ src/tclx/ucbsrc/tclbasic.sed	2022-05-16 20:24:56.000000000 -0500
@@ -1,3 +1,6 @@
+/#include "tclint.h"/a\
+\#include "tclxtend.h"
+
 /(\*cmdPtr->proc)/a\
 \        /*\
 \         * Signal handling added for Extended Tcl.\
